modules/core/include/opencv2/core/hal/intrin_wasm.hpp

```diff
@@ -0,0 +1,5045 @@
+    v_int32x4() : val(wasm_i32x4_splat(0)) {}
+    explicit v_int32x4(v128_t v) : val(v) {}
+    v_int32x4(int v0, int v1, int v2, int v3)
+    {
+        int v[] = {v0, v1, v2, v3};
+        val = wasm_v128_load(v);
+    }
+    int get0() const
+    {
+        return wasm_i32x4_extract_lane(val, 0);
+    }
+
+    v128_t val;
+};
+
+struct v_float32x4
+{
+    typedef float lane_type;
+    typedef v128_t vector_type;
+    enum { nlanes = 4 };
+
+    v_float32x4() : val(wasm_f32x4_splat(0)) {}
+    explicit v_float32x4(v128_t v) : val(v) {}
+    v_float32x4(float v0, float v1, float v2, float v3)
+    {
+        float v[] = {v0, v1, v2, v3};
+        val = wasm_v128_load(v);
+    }
+    float get0() const
+    {
+        return wasm_f32x4_extract_lane(val, 0);
+    }
+
+    v128_t val;
+};
+
+struct v_uint64x2
+{
+    typedef uint64 lane_type;
+    typedef v128_t vector_type;
+    enum { nlanes = 2 };
+
+#ifdef __wasm_unimplemented_simd128__
+    v_uint64x2() : val(wasm_i64x2_splat(0)) {}
+#else
+    v_uint64x2() : val(wasm_i32x4_splat(0)) {}
+#endif
+    explicit v_uint64x2(v128_t v) : val(v) {}
+    v_uint64x2(uint64 v0, uint64 v1)
+    {
+        uint64 v[] = {v0, v1};
+        val = wasm_v128_load(v);
+    }
+    uint64 get0() const
+    {
+#ifdef __wasm_unimplemented_simd128__
+        return (uint64)wasm_i64x2_extract_lane(val, 0);
+#else
+        uint64 des[2];
+        wasm_v128_store(des, val);
+        return des[0];
+#endif
+    }
+
+    v128_t val;
+};
+
+struct v_int64x2
+{
+    typedef int64 lane_type;
+    typedef v128_t vector_type;
+    enum { nlanes = 2 };
+
+#ifdef __wasm_unimplemented_simd128__
+    v_int64x2() : val(wasm_i64x2_splat(0)) {}
+#else
+    v_int64x2() : val(wasm_i32x4_splat(0)) {}
+#endif
+    explicit v_int64x2(v128_t v) : val(v) {}
+    v_int64x2(int64 v0, int64 v1)
+    {
+        int64 v[] = {v0, v1};
+        val = wasm_v128_load(v);
+    }
+    int64 get0() const
+    {
+#ifdef __wasm_unimplemented_simd128__
+        return wasm_i64x2_extract_lane(val, 0);
+#else
+        int64 des[2];
+        wasm_v128_store(des, val);
+        return des[0];
+#endif
+    }
+
+    v128_t val;
+};
+
+struct v_float64x2
+{
+    typedef double lane_type;
+    typedef v128_t vector_type;
+    enum { nlanes = 2 };
+
+#ifdef __wasm_unimplemented_simd128__
+    v_float64x2() : val(wasm_f64x2_splat(0)) {}
+#else
+    v_float64x2() : val(wasm_f32x4_splat(0)) {}
+#endif
+    explicit v_float64x2(v128_t v) : val(v) {}
+    v_float64x2(double v0, double v1)
+    {
+        double v[] = {v0, v1};
+        val = wasm_v128_load(v);
+    }
+    double get0() const
+    {
+#ifdef __wasm_unimplemented_simd128__
+        return wasm_f64x2_extract_lane(val, 0);
+#else
+        double des[2];
+        wasm_v128_store(des, val);
+        return des[0];
+#endif
+    }
+
+    v128_t val;
+};
+
+namespace fallback
+{
+/** @addtogroup core_hal_intrin
+
+"Universal intrinsics" is a types and functions set intended to simplify vectorization of code on
```

Comment:
Platform-specific SIMD intrinsics are not documented (only generic C++ should be documented).

So content of this file should be under documentation guard:
```
//! @cond IGNORED
... file content ...
//! @endcond
```

Please remove this comment block if nothing has been changed here. Lets avoid duplication and keep one source of origin.
The same note is for documentation blocks below. Left only unique information.

Reply:
Got it! Thanks!